### DISCUSSION TITLE
integrations/comet/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14783] [Docs]
# integrations/comet/

Learn to simplify the logging of YOLOv8 training with Comet ML. This guide covers installation, setup, real-time insights, and custom logging.

https://docs.ultralytics.com/integrations/comet/

**Comment by lauravchrobak** [comment]
I am getting this warning when using the above workflow, 'comet_ml.init() is deprecated and will be removed soon. Please use comet_ml.login()' perhaps the above should be updated to use comet_ml.login() instead?

**Reply by pderrenger** [reply]
@lauravchrobak thank you for pointing that out. The warning indicates that `comet_ml.init()` is deprecated and should be replaced with `comet_ml.login()`. We will update the documentation accordingly. For now, please use `comet_ml.login()` to avoid the deprecation warning. If you encounter any further issues, ensure you are using the latest versions of the packages. For more details, you can visit the [Comet ML integration guide](https://docs.ultralytics.com/integrations/comet/).

**Reply by glenn-jocher** [reply]
@lauravchrobak I've created a PR for this in https://github.com/orgs/ultralytics/discussions/14783

**Reply by lauravchrobak** [reply]
Wonderful, thanks for being so responsive!

**Reply by glenn-jocher** [reply]
Thank you for your feedback! We've updated our documentation to use `comet_ml.login()` instead of the deprecated `comet_ml.init()`. This should resolve the warning you're encountering. If you have any other questions or need further assistance, feel free to ask. For more details, you can visit the [Comet ML integration guide](https://docs.ultralytics.com/integrations/comet/).
